core/common/hexutil.cpp

```diff
@@ -17,6 +17,8 @@ OUTCOME_CPP_DEFINE_CATEGORY(kagome::common, UnhexError, e) {
       return "Input contains odd number of characters";
     case UnhexError::VALUE_OUT_OF_RANGE:
       return "Decoded value is out of range of requested type";
+    case UnhexError::WRONG_FORMAT:
+      return "Input data has wrong format";
```

Comment:
Too general naming.
It is used only in unhexWith0x, so we can rename to `MISSING_0X_PREFIX`, `"Missing expected 0x prefix"`